sematic/db/models/mixins/json_encodable_mixin.py

```diff
@@ -55,6 +56,23 @@ def field_from_json_encodable(cls, column: Column, json_encodable):
 ENUM_KEY = "enum"
 REDACTED_KEY = "redacted"
 ALIAS_KEY = "alias"
+CONTAIN_FILTER_KEY = "contain_filter"
+
+
+def json_string_list_contains(column: Column, value: Any) -> ColumnElement:
```

Comment:
Awkwardly, the `tags` column is represented in the DB as an actual *string* (with the value being the json-encoding of the list of tag strings), so to perform a `contains` check on it, we have to do this.